### ISSUE TITLE
Add feedback option to email us

### ISSUE THREAD

**Issue by AlistairNWard** [tonydisera/gene.iobio#396]

**Comment by tonydisera** [comment]
Al, I'm adding the feedback button to this release.  If the globals.js doesn't have feedbackEmails variable initialized, the feedback button is hidden.  For production, we need to keep this blank until the backend emailServer.js is deployed to the iobio production instances.

**Comment by AlistairNWard** [comment]
1. The screenshot in the feedback form is a little odd. Things in the screenshot can be selected and it updates the gene.iobio window from which the feedback was launched. Would it make more sense to have the screenshot be a static representation of gene.iobio when feedback was clicked? Maybe this whole section should be removed for now on account of privacy. We could allow people to attach a file, so they can send screenshots, but have to manually create the image, removing security issues from us?

2. I just submitted test feedback, but haven't received an email. I'm sure I accepted an invitation to receive these emails, so I was wondering if this feature is currently sending out emails?

**Comment by tonydisera** [comment]
Hi Al.  Thanks for testing this feature.
- There is a new gmail account gene.iobio.feedback@gmail.com that I have set up for general feedback.  I just sent a request so that you are forwarded these emails.  So after you send in the confirmation code, can you test this again?  (It worked for me, but there is no email sent from you in gene.iobio.feedback's emails from this morning.)

- Yeah, the screen capture is a home brew.  I guess that you are overlooking the quirkiness and favoring polish instead.  Well too bad.  

- The Sample names are scraped out of the screen-capture, so there shouldn't be any identifying info.  For site installs, there are 2 global variables -- feedbackAttachScreenCapture, feedbachShowURL -- that can be set to keep this out of the feedback email.  Perhaps for now, there should be a checkbox 'Show screen capture' that you have to include it disclaimer or warning.  Want to send that to legal?  Oh yeah, that's you. 

- Seriously, what we want is a screen capture plug-in.  The are out there and are not horribly expensive.  https://www.quora.com/What-is-the-best-website-feedback-widget-available-for-free

**Comment by AlistairNWard** [comment]
Yeah, I got this invite a while back and clicked the link, but I guess it didn't work. I got the new request and was told I would get forwarding and I send another test email a while ago. I got an email to me thanking me for the feedback, but never got the forwarded email.

**Comment by tonydisera** [comment]
I just re-realized (ha) that gmail accounts can only be forwarded to one email.  Should we toss a coin?  Ha.  Actually, we can add our emails to globalsDeployment.js.  Try this:

var feedbackEmails              = "gene.iobio.feedback@gmail.com,alistairnward@gmail.com";

**Comment by AlistairNWard** [comment]
Fixed.
